rest-client/src/main/java/org/sagebionetworks/bridge/rest/AuthenticationHandler.java

```diff
@@ -54,7 +56,11 @@ public AuthenticationHandler(SignIn signIn, UserSessionInfoProvider userSessionI
         }
         tryCount++;
 
-        userSession = userSessionInfoProvider.retrieveSession(signIn);
+        try {
+            userSession = userSessionInfoProvider.retrieveSession(signIn);
+        } catch (ConsentRequiredException e) {
+            LOG.debug("Ignoring 412 during auto-authentication");
```

Comment:
Okay, you're right, a 412 should interrupt whatever the user or background thread is doing, until consent is obtained. Thanks, I'll make the change!